src/LPPL-1.3c.xml

```diff
@@ -336,24 +327,26 @@
          also a statement that the distribution and/or modification of that component is constrained by the
          conditions in this license.</p>
       <p>Here is an example of such a notice and statement:</p>
-      <p>%% pig.dtx 
-        <br/>%% Copyright 2005 M. Y. Name 
-        <br/>% 
-        <br/>% This work may be distributed and/or modified under the 
-        <br/>% conditions of the LaTeX Project Public License, either version 1.3 
-        <br/>% of this license or (at your option) any later version. 
-        <br/>% The latest version of this license is in 
-        <br/>% http://www.latex-project.org/lppl.txt 
-        <br/>% and version 1.3 or later is part of all distributions of LaTeX 
-        <br/>% version 2005/12/01 or later. 
-        <br/>% 
-        <br/>% This work has the LPPL maintenance status `maintained'. 
-        <br/>% 
-        <br/>% The Current Maintainer of this work is M. Y. Name. 
-        <br/>% 
-        <br/>% This work consists of the files pig.dtx and pig.ins 
-        <br/>% and the derived file pig.sty. 
+      <standardLicenseHeader>
+      <p><optional>%%</optional> <alt name="filename" match=".+">pig.dtx</alt>
+        <br/><optional>%%</optional> Copyright <alt name="copyright" match=".+">2005 M. Y. Name</alt>
+        <br/><optional>%</optional>
+        <br/><optional>%</optional> This work may be distributed and/or modified under the
+        <br/><optional>%</optional> conditions of the LaTeX Project Public License, either version 1.3
+        <br/><optional>%</optional> of this license or (at your option) any later version.
+        <br/><optional>%</optional> The latest version of this license is in
+        <br/><optional>%</optional> http://www.latex-project.org/lppl.txt
+        <br/><optional>%</optional> and version 1.3 or later is part of all distributions of LaTeX
+        <br/><optional>%</optional> version 2005/12/01 or later.
+        <br/><optional>%</optional>
+        <br/><optional>%</optional> This work has the LPPL maintenance status "<alt name="status" match="maintained|author-maintained|unmaintained">maintained</alt>".
+        <br/><optional>%</optional>
+        <br/><optional>%</optional> This Current Maintainer of this work is <alt name="maintainer" match=".+">M. Y. Name</alt>.
```

Comment:
> `This Current Maintainer` should be `The Current Maintainer`

Hmm, that was a copy/paste error from the typo'd 1.3a.  I've pushed 2c4b18c → 8238735, fixing this and adding an `<alt>` for This/The to this part of 1.3a.  I've stuck with “This” as canonical to match the upstream 1.3a, because the distinction doesn't seem too important.  But I'm happy to make “The” the canonical 1.3a form if you'd prefer.